components/partials/swap/advanced-settings.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div>
+    <span class="block font-bold uppercase tracking-widest mb-4">
+      {{ $t('trade.swap.advanced_settings') }}
+    </span>
+    <div class="flex items-center justify-start mb-4">
+      <span class="text-gray-500 text-md">
+        {{ $t('trade.swap.slippage_tolerance') }}
+      </span>
+      <v-icon-info-tooltip
+        class="ml-1"
+        :tooltip="$t('trade.swap.slippage_tolerance_tooltip')"
+      />
+    </div>
+    <div class="grid grid-cols-7 sm:grid-cols-9 gap-2">
+      <v-button
+        v-for="(slippageTolerancePreset, i) in slippageTolerancePresets"
+        :key="i"
+        :disabled="false"
+        :ghost="false"
+        primary
+        class="rounded-md text-md h-[34px] p-0 tracking-wider col-span-2 flex items-center justify-center hover:bg-primary-500-faded hover:border hover:border-primary-600 hover:text-primary-600"
+        :class="[
+          Number(slippageTolerancePreset.value) === Number(slippageTolerance)
+            ? 'text-primary-600 bg-primary-500-faded border border-primary-600 cursor-default'
+            : 'bg-gray-700 text-gray-400',
+          {
+            'hidden sm:block': i === slippageTolerancePresets.length - 1
+          }
+        ]"
+        @click.stop="() => setSlippageTolerance(slippageTolerancePreset.value)"
```

Comment:
Let's extract this button into a separate child component, you can reference [this](https://github.com/InjectiveLabs/injective-dex/blob/dev/components/partials/markets/filters/market-type-selector.vue) for example